### PR TITLE
Feature/added title description sidebar

### LINKED ISSUE #2: Implement Basic Streamlit App Layout
**Description:**

- Develop a simple Streamlit app with a basic layout to serve as a foundation.

**Tasks:**

- Create a main page with a title and description.
- Set up navigation or sidebar (if needed).
- Ensure the app runs without errors.

### PR DESCRIPTION
**Pull request by RonanValadez35**
## Summary
This pull request introduces a proper title and description of the dashboard created by app.py. A sidebar is also implemented, but not yet functional. An image of Cisco was added as well.

## Changes
- Removed placeholders
- Added title and description
- Implemented cosmetic sidebar
- Added Cisco image to repo
 
closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # CNN 
 
+# Ronan Was Here
```
Comment (TeachMeTW) [inline README.md:3]: Remove

File: app.py [hunk 2]
```diff
@@ -1,94 +1,35 @@
...
-
-# Render the typewriter HTML in the Streamlit app
-components.html(typewriter_html, height=50)
 
+#sidebar
+st.sidebar.title("Navigation")
+page = st.sidebar.selectbox(
+    "Choose a page:",
+    ("Project Description", "User Guide", "Data")
+)
```
Comment (TeachMeTW) [inline app.py:11]: @dleiferives Do you think we should have a dropbox rather than standalone boxes?

File: app.py [hunk 3]
```diff
@@ -1,94 +1,35 @@
...
+st.title("Anamoly Detection Dashboard")
+st.markdown(
+    """
+    <div style='text-align: center;'>
+        <p>
+            This Dashboard is used to detect anomalous data. 
+            The application provides an intuitive user interface for monitoring network behavior, 
+            offering real-time insights and visualizations of detected anomalies. 
+            Users can easily upload network data, trigger analysis, and explore results through interactive dashboards and graphs. 
+            This streamlined workflow aids network administrators in identifying potential issues promptly and taking corrective actions,
```
Comment (TeachMeTW) [inline app.py:21]: not network administrators btw

**Review by TeachMeTW — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (dleiferives) [reply app.py:11]: I think that we should have standalone boxes for the moment. I expect that there will be a need for a sidebar/ side panel at some point. I imagine that mixing drop downs with a side panel would be annoying as a user. @TeachMeTW lmk if you agree, iff you disagree then I think a dropbox is should be perfect.

Comment (TeachMeTW) [reply app.py:11]: @dleiferives I agree it be too confusing, so please change this to standalone boxes in the side bar @RonanValadez35

Comment (TeachMeTW) [reply app.py:11]: Something like:
image

**Review by TeachMeTW — APPROVED** [review]

**Pull request merged**